docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md

```diff
@@ -104,7 +107,7 @@ acl purge {
 }
 ```
 
-### Enable and configure Redis Persistent
+#### Enable and configure Redis Persistent
```

Comment:
Redis + redis persistent is also Application specific

Reply:
Moved to application-specific requirements